x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -421,6 +430,27 @@ export class TimeSeriesExplorer extends React.Component {
       );
   };
 
+  loadPlotByOptions = async () => {
```

Comment:
Consider moving it outside of TimeSeriesExplorer

Reply:
Moved the logic into a separate file here https://github.com/elastic/kibana/pull/81662/commits/19e20ee5f7a61327b729440a84a8fdf6e87a6a4a